lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Read.hs

```diff
@@ -14,6 +14,9 @@ module Cardano.Wallet.Deposit.Read
     , toRawAddress
     , mockAddress
 
+    , Customer
+    , fromRawCustomer
+
```

Comment:
```suggestion
```

Wait — the concern of the `Read` modules is to read data as specified in the ledger specification, and as it appears on the blockchain, but `Customer` is not part of that. 😅  (The name `Deposit` is accidental, I want to merge this module with `Cardano.Wallet.Read` but had to choose a different name here in order to avoid a name clash.)

Could you leave the `Customer` type where it is, and instead solve the `Word32 → Customer` conversion directly in the `Cardano.Wallet.Deposit.HTTP.JSON.JSONSpec` module? 🤓

Reply:
got it - reverted the commit